packages/manager/modules/emailpro/src/service/messaging/emailpro.messaging.service.js

```diff
@@ -0,0 +1,169 @@
+import forEach from 'lodash/forEach';
+import isEmpty from 'lodash/isEmpty';
+import isString from 'lodash/isString';
+import uniqBy from 'lodash/uniqBy';
```

Comment:
When it's possible, try to replace those functions with native JS functions

Reply:
Done

Reply:
Done